build-with-docker.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+mkdir -p out
+docker build -t maruos/blueprints .
+docker run --privileged --rm \
+	-v /var/cache:/var/cache \
+        -v `pwd`:/var/maru \
```

Comment:
Please use `"$(pwd)"` syntax instead to be consistent with the other scripts. The quotes are important to avoid word splitting in Bash.